src/client/Microsoft.Identity.Client/ManagedIdentity/Constants.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    internal class Constants
+    {
+        public const string OrganizationsTenantId = "organizations";
+
+        public const string AdfsTenantId = "adfs";
+
+        // TODO: Currently this is piggybacking off the Azure CLI client ID, but needs to be switched once the Developer Sign On application is available
```

Comment:
do we have a tracking item for this?

Reply:
I will remove the unused constants here once all the apps are implemented. I just copy pasted this class.